.gitignore

```diff
@@ -1,2 +1,9 @@
 build/
 node_modules/
+Debug/
+Release/
+*.lock
+
```

Comment:
Maybe you could add the following entries to `.gitignore`:
- *.log 
- npm-debug.log*